lib/elasticsearch/config.js

```diff
@@ -74,20 +74,20 @@ const SEARCH_SCOPES = {
 }
 
 const FILTER_CONFIG = {
-  recordType: { operator: 'match', field: 'recordTypeId', repeatable: true },
-  owner: { operator: 'match', field: 'items.owner_packed', repeatable: true, path: 'items' },
-  subjectLiteral: { operator: 'match', field: 'subjectLiteral_exploded', repeatable: true },
-  holdingLocation: { operator: 'match', field: 'items.holdingLocation_packed', repeatable: true, path: 'items' },
-  buildingLocation: { operator: 'match', field: 'buildingLocationIds', repeatable: true },
-  language: { operator: 'match', field: 'language_packed', repeatable: true },
-  materialType: { operator: 'match', field: 'materialType_packed', repeatable: true },
-  mediaType: { operator: 'match', field: 'mediaType_packed', repeatable: true },
-  carrierType: { operator: 'match', field: 'carrierType_packed', repeatable: true },
-  publisher: { operator: 'match', field: 'publisherLiteral.raw', repeatable: true },
-  contributorLiteral: { operator: 'match', field: 'contributorLiteral.raw', repeatable: true },
-  creatorLiteral: { operator: 'match', field: 'creatorLiteral.raw', repeatable: true },
-  issuance: { operator: 'match', field: 'issuance_packed', repeatable: true },
-  createdYear: { operator: 'match', field: 'createdYear', repeatable: true },
+  recordType: { operator: 'match', field: ['recordTypeId'], repeatable: true },
+  owner: { operator: 'match', field: ['items.owner_packed'], repeatable: true, path: 'items' },
```

Comment:
Doesn't need to happen as part of this, but because this enables matching on multiple fields, you could change these `_packed` configs to match explicitly on each entity subfield. For example, above could use field `['items.owner.id', 'items.owner.label']`, since that's how the clause ultimately manifests, right?

Reply:
yess I like that. I knew there was something there because their structure was so similar.